Source/FikaAmazonAPI/Services/FeedService.cs

```diff
@@ -171,30 +167,14 @@ public async Task<CreateFeedDocumentResult> CreateFeedDocumentAsync(ContentType
         /// <param name="xml"></param>
         /// <param name="feedType"></param>
         /// <returns></returns>
-        public async Task<string> SubmitFeedAsync(string XmlContentOrFilePath, FeedType feedType, List<string> marketPlaceIds = null, FeedOptions feedOptions = null, ContentType contentType = ContentType.XML)
+        public async Task<string> SubmitFeedAsync(string feedContentOrFilePath, FeedType feedType, List<string> marketPlaceIds = null, FeedOptions feedOptions = null, ContentType contentType = ContentType.XML)
         {
-
             //We are creating Feed Document
             var feedCreate = CreateFeedDocument(contentType);
 
             //Uploading encoded invoice file
```

Comment:
we cant keep it only file , many others developer use this library and pass it as text XML , we need to find another solution